docs/topics/mta-cli-args.adoc

```diff
@@ -2,137 +2,162 @@
-[cols="40%,60%a",options="header",]
-|====
-|Flag |Description
+a|`-i`, `--input`
+|String
+a|Flag for the path to application source code or a binary. For more details, see xref:cli-input-argument_cli-guide[Specifying the input].
 
-| --analyze-known-libraries
-|  analyze known open source libraries
+a|`--jaeger-endpoint`
+|String
+|Flag for the jaeger endpoint to collect traces.
 
-| -h, --help
-| help for analyze
+a|`--json-output`
+|
+a|Flag to create analysis and dependency output as JSON.
 
-| -i, --input string
-|  path to application source code or a binary
+a|`-l`, `--label-selector`
+|String
+|Flag to run rules based on a specified label selector expression.
 
-| --json-output
-| create analysis and dependency output as JSON
+a|`--list-providers`
+|
+|Flag to list available supported providers.
 
-| --list-sources
-| list rules for available migration sources
+a|`--list-sources`
+|
+a|Flag to list rules for available migration sources.
 
-| --list-targets
-| list rules for available migration targets
+a|`--list-targets`
+|
+|Flag to list rules for available migration targets.
 
-| -l, --label-selector string
-| run rules based on specified label selector expression
+a|`--maven-settings`
+|string
+|Flag path to a custom Maven settings file to use
 
-| --maven-settings string
-| path to a custom maven settings file to use
+a|`-m`, `--mode`
+|String
+a|Flag for analysis mode, this must be one of `full`, for `source` and `dependencies`, or `source-only` (default `full`).
 
-| --overwrite
-| overwrite output directory
+a|`--no-proxy`
+|String
+|Flag to excluded URLs from passing through any proxy (relevant only with proxy)
 
-| --skip-static-report
-|  do not generate static report
+a|`-o`, `--output`
+|String
+|Flag for the path to the directory for analysis output. For more details, see xref:cli-output-argument_cli-guide[Specifying the output directory].
 
-| -m, --mode string
-| analysis mode" `full` (default) or `source-only`
+a|`--override-provider-settings`
+|String
+|Flag to override the provider settings. The analysis pod runs on the host network, and no providers are started.
 
-| -o, --output string | path to the directory for analysis output
+a|`--overwrite`
+|
+|Flag to overwrite the output directory. If you do not specify this argument and the `--output` directory exists, you are prompted to choose whether to overwrite the contents.
 
-| --rules stringArray | filename or directory containing rule files
+a|`--provider`
+|String Array
+|Flag to specify which provider or providers to run.
 
-| --skip-static-report
-| do not generate static report
+a|`--rules`
+|String Array
+a|Flag to specify the filename or directory containing rule files. Use multiple times for additional rules, for example, `--rules --rules …`.
 
-| -s, --source string
-| source technology to consider for analysis. To specify multiple sources, repeat the parameter: --source <source_1> --source <source_2> etc.
+a|`--run-local`
```

Comment:
Just note, this is default `true` in 7.2, can be disabled with `--run-local=false`.

Reply:
Damn it! Will have to do a second PR for that